src/NativeScript/Calling/FFICall.cpp

```diff
@@ -8,9 +8,94 @@
 
 #include "FFICall.h"
 
+const size_t MAX_KNOWN_ALIGNMENT = 8;
+
 namespace NativeScript {
 using namespace JSC;
 
+FFICall::FFICallFrame::FFICallFrame(FFISignature& signature, ExecState* execState, Byte* buffer)
+    : _execState(execState) {
+
+    _result = buffer + signature.resultOffset;
+    _arguments = reinterpret_cast<void**>(buffer + signature.argsArrayOffset);
+
+    for (size_t i = 0; i < signature.argsCount; ++i) {
+        _arguments[i] = buffer + signature.argValueOffsets[i];
+    }
+}
+
+FFICall::FFISignature::FFISignature()
+    : argValueOffsets(nullptr)
+    , postCalls(nullptr) {
+}
+
+void align_offset(size_t& offset, size_t alignment) {
```

Comment:
You can use `static` (or anonymous C++ namespaces) to make functions and global variables internal to the file.